Gemfile.lock

```diff
@@ -33,6 +33,10 @@ GEM
       activemodel (= 5.0.1)
       activesupport (= 5.0.1)
       arel (~> 7.0)
+    activerecord-postgres-earthdistance (0.5.2)
```

Comment:
I see some of these Gems were updated to different versions--let's try not to use `bundle update` because it can cause conflicts with everyone else's local Gem dependencies. Of course there will be times when we might need to, but let's talk about it first. :) (I know this is so different from npm land where people update versions constantly but for some reason Ruby Gems are much more finicky!)